endpoints/openrtb2/sample-requests/valid-whole/exemplary/bidadjustments-cpm.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Wow! Changes and local testing look good. 
Just an idea: do you think it makes sense to move bidadjustments json tests to a separate directory `sample-requests/bidadjustments`?